sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ConnectionStringProperties.java

```diff
@@ -21,6 +29,8 @@
     private static final String ERROR_MESSAGE_FORMAT = "Could not parse 'connectionString'. Expected format: "
         + "'Endpoint={endpoint};SharedAccessKeyName={sharedAccessKeyName};"
         + "SharedAccessKey={sharedAccessKey};EntityPath={eventHubName}'. Actual: %s";
```

Comment:
Not the code change of this PR. But eventHubName might mislead users. It can be either an eventhub or servicebus it's it's an azure-core class.
And EntityPath is optional in the endpoint.

Reply:
`entityPath` is common across both. I will use that.